tests/README.md

```diff
@@ -0,0 +1,18 @@
+Tests are currently available for two different software token implementations,
+NSS's softokn and softhsm (with some limitations).
+
+The easiest way to configure things to run manual tests is to simply execute next command from project's root directory:
+```bash
+meson test -C builddir
+```
+Be aware that previous command might change if a different build directory has been selected. Previous command will create two temporary directories with each token's configurations.
```

Comment:
Please rephrase the first sentence as:
`The 'builddir' argument is the name of the build directory, if you built in another directory you'll have to provide the name of the build directory here.
This command will create a temporary directory for each token being tested.`

(I avoid saying two so we do not need to remember to fix this part of the README when we add another token soon).